packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -28,6 +34,14 @@ async function initDefaultWorkflow({ workflowsService, stagesService }) {
 
 function extendReviewWorkflowContentTypes({ strapi }) {
   const extendContentType = (contentTypeUID) => {
+    const assertContentTypeCompatibility = (contentType) =>
+      contentType.collectionName.length <= MAX_CONTENT_TYPE_NAME_LEN;
+    const incompatibleContentTypeAlert = (contentType) => {
+      strapi.log.warn(
+        `Content type "${contentType.info.displayName}" cannot have Review Workflow activated as the name is too long. (${MAX_CONTENT_TYPE_NAME_LEN} maximum characters)`
```

Comment:
Maybe it's me, but this chatgpt alternative seems clearer to me. wdyt?
```suggestion
        `Review Workflow cannot be activated for the content type with the name '${contentType.info.displayName}' because the name exceeds the maximum length of ${MAX_CONTENT_TYPE_NAME_LEN} characters.`
```